ci/matrix.xml

```diff
@@ -61,6 +61,7 @@
         'set scalaJSOptimizerOptions in testingExample ~= (_.withDisableOptimizer(true))' \
         ++$scala testingExample/test:run testingExample/test &&
     sbtretry ++$scala testSuiteJVM/test testSuiteJVM/clean &&
+    sbtretry ++$scala testSuite/test:runMain org.scalajs.testsuite.junit.JUnitBootstrapTest testSuite/clean &&
```

Comment:
As the CI failure highlights, there should be `'`s around `testSuite/test:runMain org.scalajs.testsuite.junit.JUnitBootstrapTest`. So this line should be:
```
    sbtretry ++$scala 'testSuite/test:runMain org.scalajs.testsuite.junit.JUnitBootstrapTest' testSuite/clean &&
```